cmake/FindClang.cmake

```diff
@@ -96,6 +97,7 @@ if(CLANG_CXX)
   _Clang_find_and_add_cxx_lib(LLVMDemangle llvm/Demangle/Demangle.h)
 endif()
 
+_Clang_find_program(Clang_FORMAT clang-format)
```

Comment:
This finds the clang-format in the downloaded clang directory, right?

Reply:
it finds system clang-format if SYSTEM_CLANG is enabled, downloaded clang-format otherwise